packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -782,6 +770,20 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
         }
       }
 
+      const maybeUpdatedQuoteState = S.getBuyQuote(yield select()).getOrFail(BS_ERROR.NO_QUOTE)
```

Comment:
As descriptive as it can get 😀

We still refreshing quotes while user is interacting with gpay applepay - so at this point quote state *might* be different from the one in the payload.